src/components/rules/rules.scss

```diff
@@ -0,0 +1,43 @@
+.box {
+  margin: 0 auto;
+  width: 60vw;
+  height: 60vh;
+  overflow: auto;
+  text-align: center;
+  font-size: 1.3em;
+  line-height: 1.5;
+  padding-right: 3%;
+  color: #fff;
+
+  & p {
+    margin: 1.5em 0 0;
+  }
+
+  & p:first-child {
+    margin: 0;
+  }
```

Comment:
That's not how BEM works. We do not use raw tags in our scss code unless you have a really good reason.